docs/reference/runners.md

```diff
@@ -0,0 +1,23 @@
+# Runners
+
+|                         Provider                          |                         Platforms                          | CPU | GPU | RAM  |   Disk    | Time  | Concurrent jobs |                    Notes                     |
+| :-------------------------------------------------------: | :--------------------------------------------------------: | :-: | :-: | :--: | :-------: | :---: | :-------------: | :------------------------------------------: |
+| [Azure Pipelines][azure] ([Microsoft-hosted][ms-hosted])  | `linux-64`, `win-64`, `linux-aarch64`†‡, `linux-ppc64le`†‡ |  2  | No  | 7GB  | 14GB SSD˟ |  6h   |       100       |          [More details][ms-hosted]           |
+|                             ↳                             |                   `osx-64`, `osx-arm64`‡                   |  3  |     | 14GB |           |       |                 |
+| [Github Actions][gh-actions] ([Github-hosted][gh-hosted]) | `linux-64`, `win-64`, `linux-aarch64`†‡, `linux-ppc64le`†‡ |  2  | No  | 7GB  | 14GB SSD˟ |  6h   |       100       | Reserved for internal conda-forge usage only |
+|                             ↳                             |                   `osx-64`, `osx-arm64`‡                   |  3  |     | 14GB |           |       |                 |
+|                    [Travis CI][travis]                    |              `linux-aarch64`, `linux-ppc64le`              |  2  | No  | ~4GB |   ~18GB   | 50min |        ?        |
```

Comment:
is this up to date to what the actual CI's runners are currently using?

Reply:
Yea, mostly.